src/hardware/addon-protocols/rs232-usb.md

```diff
@@ -231,6 +231,25 @@ Issued by the GO device on receipt of Binary Data of 256 bytes or more from the
 | Checksum | 2 | 3+x |
 | ETX (0x03) | 1 | 5+x |
 
+### Msg Type 0x26: Protobuf data packet
+
+Issued by the GO device in response to Msg Type 0x8C.
+Information published by the GO device.
+
+### Msg Type 0x27: Add-On protocol version to external device
+
+Issued by the GO device on receipt of 0x8B.Should be paired with 0x8B. 0x27 is a reply to 0x8B.
```

Comment:
I like the information on who issues the command. Can we use IOX instead of Go device in this context? It is so confusing when sometimes we regards IOX as part of GO, and sometimes treated them separately. It seems to be a message between external device and IOX, nothing to do with GO, or?

Reply:
yes, for the code level , it is right, it is from iox.  However, I am thinking about user friend syntax. For thirdparty,  Go device and usb like a bundle, they will used it together. 
So user actually might be confused, since there is no Section "Messages from iox".  for user, etiher from iox or go is from the same place. what they really want is   "Messages from external device." or "Messages form Go"
Thanks. 
https://docs.google.com/document/d/1SdDf78K7OUVpMWRelSwTW_s3kfAgGiWAhRXUhpTKMdg/edit#heading=h.i74exiijwj50